events/templates/prompts_for_sessions.md

```diff
@@ -0,0 +1,28 @@
+# Prompts for sessions in quarterly meetings
+
+At the quarterly meetings we ask registrants whether they would like to host a `Project workshop` or `Breakout discussion` on the day.
+
+Those who say yes are sent the following prompts to gather info for their sessions. 
+This is sent to them, and organised, via email.
+
+## Project workshops
+
```

Comment:
```suggestion

Project workshops are focused discussions on a specific project, issue or challenge.
They are intended to serve as a collaborative work sessions where the community helps the presenter to move forward in their project. 

```

Reply:
I think that while we figure out an intuitive terminology we should add details about how we envision each type of breakout